test-serialization/src/lib.rs

```diff
@@ -28,25 +30,48 @@ mod tests {
                 .unwrap()
                 .bin_prot_rule
         };
+        static ref TEST_BLOCK_1: bin_prot::Value = load_test_block();
     }
 
     #[test]
-    fn test_all_block_subtypes() {
-        let mut de = Deserializer::from_reader_with_layout(BLOCK_BYTES, &BLOCK_RULE);
-        let block: bin_prot::Value =
-            Deserialize::deserialize(&mut de).expect("Failed to deserialize block");
+    fn test_protocol_state_body_constants() {
+        for block in [&TEST_BLOCK_1] {
+            test_in_block::<Length>(block, &["t/protocol_state/t/t/body/t/t/constants/t/t/k"]);
+            test_in_block::<Length>(
+                block,
+                &["t/protocol_state/t/t/body/t/t/constants/t/t/slots_per_epoch"],
+            );
+            test_in_block::<Length>(
+                block,
+                &["t/protocol_state/t/t/body/t/t/constants/t/t/slots_per_sub_window"],
+            );
```

Comment:
We can combine paths that share the same type 
```suggestion
        test_in_block::<Length>(&block, &[
            "t/protocol_state/t/t/body/t/t/constants/t/t/k",
            "t/protocol_state/t/t/body/t/t/constants/t/t/slots_per_epoch",
            "t/protocol_state/t/t/body/t/t/constants/t/t/slots_per_sub_window"
        ]);
```